README.md

```diff
@@ -172,6 +172,8 @@ Mona uses several environment variables you can set as you prefer:
   from the message dict to export (default value: False). Note that passing a None value may be required in order to delete a pre-existing 
   value. To allow None values, use filter_none_fields=False which overrides this parameter both in export() and 
   export_batch() functions.
+- MONA_SDK_DEFAULT_SAMPLING_FACTOR - A float in the range [0, 1], which sets the random client-side sampling done by the SDK before sending the data into Mona servers. If this value is less than 1, only a random** sample of the given proportion is actually going to be sent, leaving the rest of the data unattended. Use with caution. (random** - using hashing with sha224 on the context id, if supplied, or by random.random() otherwise.)
```

Comment:
@nemo-mona I think the use of double asterisk here is confusing... It's creating bold in MD readers like here and should be avoided.